keyboards/ploopyco/trackball/trackball.c

```diff
@@ -132,16 +145,27 @@ __attribute__((weak)) void process_mouse(report_mouse_t* mouse_report) {
 }
 
 bool process_record_kb(uint16_t keycode, keyrecord_t* record) {
-    if (debug_mouse) {
-        dprintf("KL: kc: %u, col: %u, row: %u, pressed: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed);
+    if (true) {
+        xprintf("KL: kc: %u, col: %u, row: %u, pressed: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed);
     }
 
     // Update Timer to prevent accidental scrolls
-    if ((record->event.key.col == 2) && (record->event.key.row == 0)) {
+    if ((record->event.key.col == 1) && (record->event.key.row == 0)) {
         lastMidClick = timer_read();
         is_scroll_clicked = record->event.pressed;
     }
 
+    if (!process_record_user(keycode, record)) { return false; }
+
+    if (keycode == DPI_CONFIG && record->event.pressed) {
+        keyboard_config.dpi_config++;
+        if (keyboard_config.dpi_config > DPI_OPTION_SIZE) {
+            keyboard_config.dpi_config = 0;
+        }
```

Comment:
```suggestion
        keyboard_config.dpi_config = (keyboard_config.dpi_config + 1) % DPI_OPTION_SIZE;
```
Not 100% sure this is correct but you get the idea.

Reply:
Looks correct, yeah.  Just not as easy to read, IMO.

Reply:
Also, couldn't you just do: 
```c
        keyboard_config.dpi_config = (keyboard_config.dpi_config++) % DPI_OPTION_SIZE;
```

And if not that, then:
```c
        keyboard_config.dpi_config = (++keyboard_config.dpi_config) % DPI_OPTION_SIZE;
```

Reply:
Probably the latter, yes.